packages/shared/src/server/definitions.ts

```diff
@@ -172,3 +172,153 @@ export const convertScoreReadToInsert = (
     event_ts: new Date(record.event_ts).getTime(),
   };
 };
+
+/**
+ * Expects a single record from a `select * from traces` query. Must be a raw query to keep original
+ * column names, not the Prisma mapped names.
+ */
+export const convertPostgresTraceToInsert = (
```

Comment:
Logic is fully copied from the migration scripts. That way we have the same mapping in the ingestion pipeline.

Reply:
very nice!